src/main/java/com/management/HumanResources/service/ParseService.java

```diff
@@ -78,6 +71,9 @@ public EmployeeTime jsonToEmployeeTime(JSONObject obj) {
         String[] employeeTimeOffCsvs = psv.split("\\|");
         List<TimeOff> timeOffs = new ArrayList<TimeOff>();
         for (String employeeTimeOffCsv : employeeTimeOffCsvs) {
+            if(employeeTimeOffCsv.equals("null")){
```

Comment:
No need to check each CSV. You could just check if `psv.equals("null")` on the first line of the method.

Reply:
Great Idea!